charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -355,12 +426,113 @@ defaultCRConfig:
 # This configuration allows for multiple layers of overrides for different clusters. For example, you could
 # create a collector called test with an OTLP exporter in your values.yaml, and then override the endpoint's
 # destination in a file called values-staging.yaml.
-collectors: {}
+collectors:
+  daemon:
+    name: daemon
```

Comment:
should we use the chart name/release name as part of these collectors?

Reply:
that's automatically prefixed, this is the suffix... maybe i should change this to be "suffix" and then expose a fullname override